assets/helpers/url.js

```diff
@@ -1,5 +1,19 @@
 // @flow
 
+// ----- Routes ----- //
+
+const routes = {
```

Comment:
I don't think the routes belong in a file with url parsing tools.  `routes.js` maybe?